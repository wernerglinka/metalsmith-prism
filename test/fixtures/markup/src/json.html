<code class="language-json">
{
  "name": "metalsmith-prism",
  "description": "Provides syntax highlighting"
}
</code>
