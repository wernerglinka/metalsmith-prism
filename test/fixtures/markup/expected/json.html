<html><head></head><body><code class="language-json">
  <span class="token punctuation">{</span>
    <span class="token property">"name"</span><span class="token operator">:</span> <span class="token string">"metalsmith-prism"</span><span class="token punctuation">,</span>
    <span class="token property">"description"</span><span class="token operator">:</span> <span class="token string">"Provides syntax highlighting"</span>
  <span class="token punctuation">}</span>
</code>
</body></html>
